http://bugs.python.org/issue10910
